Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM openjdk:11
+COPY . /usr/src/main
+WORKDIR /usr/src/main
+RUN javac Hello.java
```

Comment:
this is not correct docker example : please have a look : https://spring.io/guides/topicals/spring-boot-docker/

Reply:
Will make the changes Manzarul